docs/GettingStarted.md

```diff
@@ -387,6 +387,38 @@ giving precedence to the middleware settings. Inherited configurations are:
 * ``trace_agent_hostname``
 * ``trace_agent_port``
 
+### Faraday
+
+The `faraday` integration is available through the `ddtrace` middleware:
+
+    require 'faraday'
+    require 'ddtrace'
+
+    Datadog::Monkey.patch_module(:faraday) # this will simply register our middleware
+
+    connection = Faraday.new('https://example.com') do |builder|
+      builder.use(:ddtrace, options)
```

Comment:
* do they need to add `builder.use(:ddtrace, options)` all the time? what if they have 100 connections to 100 domains? they need to change all these blocks?
* what about `options`? if ^^ is true, it means they also need to find a way to propagate the options someway

Reply:
yes, that's how the `faraday` middleware API works. it's a common pattern to wrap this instantiation on a helper method, but I believe this is not something we should address, since it's simply the way the gem was meant to be.

Reply:
I'm wondering if it has a huge code impact (in the sense of amount of changes). Let see how it plays with real projects. We can still build something on top of that if it's required.